packages/frontend-2/lib/workspaces/graphql/queries.ts

```diff
@@ -0,0 +1,22 @@
+import { graphql } from '~~/lib/common/generated/gql'
+
+export const workspacePageQuery = graphql(`
+  query WorkspacePageQuery(
+    $workspaceId: String!
+    $filter: WorkspaceProjectsFilter
+    $cursor: String
+  ) {
+    workspace(id: $workspaceId) {
+      id
```

Comment:
Where's id and role coming from? And totalCount & cursor under projects? Is there a specific reason they're not covered by fragments?